R/getPrevalence.R

```diff
@@ -109,12 +115,20 @@
 #' # Note that the data (GlobalPatterns) is here in absolute counts
 #' # (and not compositional, relative abundances)
 #' # Prevalence threshold 50 percent (strictly greater by default)
-#' taxa <- getPrevalentTaxa(GlobalPatterns,
-#'                          rank = "Phylum",
-#'                          detection = 1/100,
-#'                          prevalence = 50/100,
-#'                          as_relative = TRUE)
-#' head(taxa)
+#' prevalent <- getPrevalentTaxa(GlobalPatterns,
+#'                               rank = "Phylum",
+#'                               detection = 10,
+#'                               prevalence = 50/100,
+#'                               as_relative = FALSE)
+#' head(prevalent)
+#' 
+#' # Gets a subset of object that includes prevalent taxa
+#' x_prevalent <- subsetByPrevalentTaxa(GlobalPatterns,
```

Comment:
In these cases you could work with `altExp` for storing the data. That would actually good practice since the data is kept in one object and makes subsetting and other task a bit easier.